docs/DeveloperGuide.md

```diff
@@ -387,11 +387,29 @@ otherwise)
 
 **MSS**
 
-1. User wants to look for items that are expiring soon.
+1. User search for items that are expiring soon.
 2. StoreMando returns a list of expiring items.
 
    Use case ends.
 
+**Extensions**
+
+* 1a. User input number is a negative number.
+
+    * 1a1. StoreMando shows an error message.
+    
+    * 1a2. StoreMando prompt the user for a correct input.
+
+      Use case resumes at step 1.
+    
+* 1a. Time unit input is neither is day(s) or week(s)
```

Comment:
Just a minor language error, "is neither day(s) or week(s)"

Reply:
All right thanks!